src/osdc/Objecter.cc

```diff
@@ -3493,6 +3508,13 @@ void Objecter::handle_osd_op_reply(MOSDOpReply *m)
   /* get it before we call _finish_op() */
   auto completion_lock = s->get_lock(op->target.base_oid);
 
+  if (mclock_service_tracker) {
+    unsigned prev_seed = ceph_stable_mod(op->target.pgid.ps(),
+	                                 op->target.pg_num,
+	                                 op->target.pg_num_mask);
+    uint32_t shard_index = prev_seed % num_shards;
+    qos_trk->track_resp(OsdID(op->target.osd, shard_index), m->get_qos_resp());
+  }
```

Comment:
@TaewoongKim, Hi~
recently I'm reading the code of your current pr, but I encountered with some difficulty of understanding about how to static io of a pool, Could you please help me? thx!
since it's a QoS base of pool, there must be somewhere to static the sum of io for a pool. I think  here is the entry of statistic delta, rho of io for a pool.
the track_resp function receive two parameter, one is serverID, another is phase. for the serverID, which is a type of pair(osd, shard_index), the target.osd is the ID of a osd, what's the meaning of shard_index? the prev_seed is equal to the num of PG?  I'm confused with the pair(osd, shard_index), how it static delta, rho related to a pool? I feel the statistic is related to osd.
I'm waiting for your reply, thx! 
My Email is : redenval@gmail.com